### PR TITLE
add support for the latest iOS backup format (.db)

### LINKED ISSUE #1: 现在是不是导不出wechat_log了

### PR DESCRIPTION
**Pull request by w1ndy**
This may also resolve #1

### CODE REVIEW

File: wechat-exporter.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def save_log(self):
         fo = bz2_open(fpath + '.csv.bz2', 'wt', encoding='utf8')
       else:
         fo = open(fpath + '.csv', 'w', encoding='utf8')
-      wt = csv_writer(fo)
+      wt = csv_writer(fo, quoting=csv_QUOTE_ALL, lineterminator='\n')
```
Comment (12425) [inline wechat-exporter.py]: Per [RFC 4180](https://tools.ietf.org/html/rfc4180#section-2), lines of a CSV should be delimited by CRLF, so the default behavior of `csv.writer` is fairly good.

**Review by 12425 — COMMENTED** [review]
Could you tell me your iOS version and WeChat version? My iPhone 4S can still work with the latest iOS and WeChat.

**Comment by w1ndy** [comment]
It's a quite while ago, but I guess I was using the latest WeChat, iTunes, and iOS 11 on iPhone 7 back then. For the CSV format, I did have some difficulties in processing the output file in Excel. Maybe we can consider using more consistent file formats like XML or JSON?

**Comment by 12425** [comment]
As I know, Excel has very limited support for CSV. Often it doesn't even recognize UTF-8. You may try another CSV viewer then. To me, a text editor is good.
XML, JSON, YAML, etc. has too much redundancy. Chat history is a table, so CSV is the best text format to save them. Or we can save in a Sqlite database later. 😄

**Comment by w1ndy** [comment]
Sounds good to me 👍

**Review by 12425 — APPROVED** [review]

**Pull request merged**